### PR TITLE
Cabol.3.cowboy swagger handler

### PR DESCRIPTION
**Pull request by cabol**
- Issue #3 solved.
- Fixed Makefile to fetch latest trails release (0.0.2).

### CODE REVIEW

File: test/cowboy_swagger_test_utils.erl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        , api_call/4
+        ]).
+
+-type config() :: proplists:proplist().
+-export_type([config/0]).
+
+-spec all(atom()) -> [atom()].
+all(Module) ->
+  ExcludedFuns = [module_info, init_per_suite, end_per_suite, group, all],
+  Exports = Module:module_info(exports),
```
Comment (gadgetci) [inline test/cowboy_swagger_test_utils.erl]: According to **Elvis**:
> Remove the dynamic function call on line 18. Only modules that define callbacks should make dynamic calls.

Comment (gadgetci) [inline test/cowboy_swagger_test_utils.erl]: According to **Elvis**:
> Remove the dynamic function call on line 18. Only modules that define callbacks should make dynamic calls.

File: test/test.config [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+ {example,
+  [
+   {http_port, 8080},
+   {http_listener_count, 10}
+  ]
+ }
+].
```
Comment (davecaos) [inline test/test.config]: I think  because that EOF gadget/dialyzer is complaining.

File: test/example_echo_handler.erl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            required => false,
+            type => <<"string">>}
+        ]
+      }
+    },
+  [trails:trail("/message/[:echo]", example_echo_handler, [], Metadata)].
+
+%% cowboy
+allowed_methods(Req, State) ->
+  {[<<"GET">>, <<"PUT">>,   <<"HEAD">>], Req, State}.
```
Comment (elbrujohalcon) [inline test/example_echo_handler.erl]: too many spaces

**Pull request merged**